Dockerfile

```diff
@@ -36,6 +36,8 @@ RUN apt-get update -q && apt-get install -q -y --no-install-recommends \
     python3-stringtemplate3 \
     spin \
     wget \
+    libncurses5 \
```

Comment:
Please note that these are in a more or less alphabetical order, for easier maintenance

Reply:
ack